### PR TITLE
Update IAudioDeviceService.cs

### PR DESCRIPTION
**Pull request by AcK77**
Stubs:
- QueryAudioDeviceSystemEvent
- GetActiveChannelCount

### CODE REVIEW

File: Ryujinx.Core/OsHle/Services/Aud/IAudioDeviceService.cs [hunk 1]
```diff
@@ -45,6 +52,9 @@ public long ListAudioDeviceName(ServiceCtx Context)
                 Position += Buffer.Length;
             }
 
+            //TODO: We shouldn't be signaling this here.
+            SystemEvent.Handle.Set();
```
Comment (gdkchan) [inline Ryujinx.Core/OsHle/Services/Aud/IAudioDeviceService.cs]: It's weird that this is signaled here. Can't you signal right after `SystemEvent = new KEvent();`? Feels less hacky this way at least :P

File: Ryujinx.Core/OsHle/Services/Aud/IAudioDeviceService.cs [hunk 2]
```diff
@@ -11,13 +12,21 @@ class IAudioDeviceService : IpcService
 
         public override IReadOnlyDictionary<int, ServiceProcessRequest> Commands => m_Commands;
 
+        private KEvent SystemEvent;
+        //TODO: We shouldn't be signaling this here.
+        SystemEvent.Handle.Set();
```
Comment (gdkchan) [inline Ryujinx.Core/OsHle/Services/Aud/IAudioDeviceService.cs]: I meant below `SystemEvent = new KEvent();`, not the variable declaration. I think this code doesn't compile as this call outside of a function is not valid code.

**Comment by AcK77** [comment]
Sorry for the mistake xD!

**Comment by gdkchan** [comment]
lgtm, thanks!

**Pull request merged**